src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,44 @@
 
 public class DefaultCountingOutRhymer {
 
-	private int[] numbers = new int[12];
+	private static final int STACK_EMPTY = -1;
 
-	public int total = -1;
+	private static final int STACK_CAPACITY = 12;
 
-	public void countIn(int in) {
+	private int[] numbers = new int[STACK_CAPACITY];
+
+	public int total = STACK_EMPTY;

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)